sts/simulator/simulator.go

```diff
@@ -36,6 +36,7 @@ func init() {
 		if r.IsVPX() {
 			path, handler := New(s.Listen, r.OptionManager().Setting)
 			s.Handle(path, handler)
+			s.Handle("/sts/system-STSService/sdk", handler)
```

Comment:
`sts.SystemPath` ?

Reply:
Hmm I think there was an import cycle. I think it's because sts/client_test.go is in the 'sts' package, not sts_test . I could look to move it to a separate package

Reply:
It's ok to leave this as-is or move `const SystemPath` to sts/internal.
Changing the client_test package is a-ok too, if not too much of a hassle.

Reply:
Fixed, was a clean change as it wasn't using anything internal to the sts package.